docs/changelog.rst

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+0.3.0 (Unreleased)
+------------------
+
+Changed
+~~~~~~~
+
+- Refactor to build upon `ipython-sql <https://pypi.org/project/ipython-sql/>`__.
+  Removes several functions that can be replaced with direct usage of ipython-sql magics in the notebook,
+  and replace all remaining sql calls with calls to ipython-sql.
+
+  Replacements (must run ``%load_ext sql`` first, and ``%config SqlMagic.autopandas = True`` to get a pandas ``DataFrame``):
+
+  * :meth:`ocdskingfishercolab.create_connection` — replaced by using an ipython-sql connection string, e.g. ``%sql postgresql://user:pass@host/db``
+  * :meth:`ocdskingfishercolab.execute_statment`, :meth:`ocdskingfishercolab.get_dataframe_from_cursor` and
+    :meth:`ocdskingfishercolab.get_dataframe_from_query` — replaced by sql magic from ipython-sql, e.g. ``%sql SELECT a FROM b``
+  * :meth:`ocdskingfishercolab.get_list_from_query` — replaced by :meth:`get_ipython_sql_resultset_from_query`. This returns an ipython-sql ResultSet, which can behave broadly like a list, but also has other features.
```

Comment:
Add link to ResultSet documentation?

Reply:
There doesn't seem to be explicit docs on the ResultSet, just [implicitly in the examples](https://pypi.org/project/ipython-sql/#examples), and a [short docstring in the code](https://github.com/catherinedevlin/ipython-sql/blob/b24ac6e9410416eafde86ae22fd8d6f34acbe05d/src/sql/run.py#L99) that doesn't get extracted anywhere.

I suggest updating the sentence to add a bit more explanation:

``` rst
This returns an `ipython-sql ResultSet <https://pypi.org/project/ipython-sql/#examples>`__, the type returned by the ``%sql%`` magic when ``autopandas`` is off. It behaves like a list, but with extra methods.
```

Reply:
Okay, sounds good.